### PR TITLE
dns.he.net added

### PR DESCRIPTION
**Pull request by mayurkr**
Free DNS Hosting service

### CODE REVIEW

**Comment by nhymxu** [comment]
I like it, I'm using it now. Fast update

File: README.md [hunk 1]
```diff
@@ -279,6 +279,7 @@ Table of Contents
   * https://www.acquia.com/free - Hosting for Drupal sites. Free tier for developers. Free development tools (such as Acquia Dev Desktop) also available.
   * https://www.bitballoon.com - BitBalloon offers hosting for static sites and apps. Free on a subdomain.
   * https://freedns.afraid.org/ - Free DNS hosting 
+  *	https://dns.he.net/ - Free DNS hosting service with Dynamic DNS Support
```
Comment (ripienaar) [inline README.md]: can you fix the spacing here please?

thanks

Comment (mayurkr) [inline README.md]: Sorry for that, fixed it now :)

**Comment by ripienaar** [comment]
thanks

**Pull request merged**